console: server-build script fails to download assets

As of https://github.com/hasura/graphql-engine/commit/eb0c1f66425568c5fcf32c882223a2e92503412d, I get the following error when I run `npm run server-build`:

```
CommandException: No URLs matched: gs://hasura-graphql-engine/console/assets/common
CommandException: 1 file/object could not be transferred.
mv: rename ./static/dist/common/css/font-awesome.min.css.gz to ./static/dist/common/css/font-awesome.min.css: No such file or directory
gzip: can't stat: ./static/dist/common/css/font-awesome.min.css (./static/dist/common/css/font-awesome.min.css): No such file or directory
make: *** [server-build] Error 1
```

Do I need credentials to access the `gs://hasura-graphql-engine` bucket? And if so, is it intentional? It seems important that the `server-build` script be accessible to OSS contributors.

--- a/console/Makefile
+++ b/console/Makefile
@@ -1,6 +1,6 @@
 export PATH := node_modules/.bin:$(PATH)
 DIST_PATH ?= ./static/dist
-BUCKET_NAME ?= hasura-graphql-engine
+BUCKET_NAME ?= graphql-engine-cdn.hasura.io
 VERSION ?= $(shell ../scripts/get-version.sh)
 
 all: deps build gzip-assets gcloud-cp-stable gcloud-set-metadata

